### PR TITLE
Inc 17/side bar

### PR DESCRIPTION
**Pull request by Viarcinski**
1.Create sidebar component
2.Create new layout with sidebar

### CODE REVIEW

File: @ui/ui-kit/Icon/MakeIcon.tsx [hunk 1]
```diff
@@ -0,0 +1,18 @@
+import { FC } from 'react'
+import { MakeIconPropsType } from './iconsType'
+import style from './IconsComponent.module.scss'
+import Link from 'next/link'
+
+export const MakeIcon: FC<MakeIconPropsType> = (props) => {
```
Comment (irinrudko) [inline @ui/ui-kit/Icon/MakeIcon.tsx:6]: you can destructure values just inside props:

export const MakeIcon: FC<MakeIconPropsType> = ({ OutlineIcon, Icon, className, text, isActive, isDisabled, url }) => {
}

**Review by irinrudko — APPROVED** [review]
left comment but it's just a suggestion. everything in the code and the page itself look nice and human-friendly! thanks!

**Review by maskpa100 — COMMENTED** [review]
ok

File: @ui/ui-kit/Icon/MakeIcon.tsx (hunk 1 above)
Comment (Viarcinski) [reply @ui/ui-kit/Icon/MakeIcon.tsx:6]: This is a good point, but in my opinion it makes the function declaration way too long. So I moved it to the next line

like this: 

export const MakeIcon: FC<MakeIconPropsType> = (props) => {
  const { OutlineIcon, Icon, className, text, isActive, isDisabled, url } = props
}

**Review by TURP1 — APPROVED** [review]
Resolve the conflicts

**Comment by TURP1** [comment]
Знімок екрана 2023-07-19 о 18 18 55
if you were to write tests for this component, one of the tests would be:
when there is a disabled state, there should be no such style as a border

**Review by maskpa100 — APPROVED** [review]

**Review by ilich01 — APPROVED** [review]

**Pull request merged**